docs/src/examples/atomix.md

```diff
@@ -0,0 +1,69 @@
+# Atomic operations with Atomix.jl
+
+In case the different kernels access the same memories, [race conditions](https://en.wikipedia.org/wiki/Race_condition) can occur.
```

Comment:
```suggestion
In case the different kernels access the same memory locations, [race conditions](https://en.wikipedia.org/wiki/Race_condition) can occur.
```